src/diffusers/models/transformer_temp.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
`temporal_transformer.py` or `transformer_temporal.py` is a more complete name IMO.

Reply:
No strong opinions, though.

Reply:
Sounds good will rename